src/test/java/formflow/library/utilities/TestUtils.java

```diff
@@ -1,19 +1,10 @@
 package formflow.library.utilities;
 
-import formflow.library.data.Submission;
 import java.io.File;
 import java.net.URL;
-import java.util.HashMap;
 
 public class TestUtils {
 
-  public static void resetSubmission() {
```

Comment:
Aren't we still using this [function in cleanup](https://github.com/codeforamerica/form-flow/blob/ec70d87a58e35c6ae3b8bdc4ff91e77bdc71e085/src/test/java/formflow/library/utilities/AbstractMockMvcTest.java#L93-L96)?

Reply:
It is being removed in the same commit.

Reply:
ahh I missed it, ty.